### DISCUSSION TITLE
New action to trigger crash and flashback

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#17] [Q&A]
Unable to enter Chinese characters in the address bar of Thorium browser to call google search, it will crash and crash stably. It is speculated that the API interface call of Google's native service and search service caused the bottom layer of the browser to crash, and although the Sync service is marked with this function, in fact, even if there is a scientific Internet connection and Google mailbox, the login fails. Please be aware. Going to Google to search through the custom homepage (I set Bing China) should not work, based on the experience of version 5145. Please build the thorium team and schedule a fix; thanks!

**Comment by gz83** [comment]
@whxuezhe Are you currently logged into your Google account at Thorium?

**Reply by whxuezhe** [reply]
@gz83 I just discovered the thorium build, and now I know that the thorium build has its own plug-ins; I don't know much about the difference between the Google accounts used for cloud synchronization and the browsers used for different builds.If possible, briefly describe the cloud sync Google account used for thorium builds, does it require a separate Google account for chrome builds to register? Thanks!

**Reply by whxuezhe** [reply]
@gz83 It should be partly related to the real-time availability of Google's native services, and that the porting of this function is not completely seamless. After two attempts, the synchronization service has been successfully enabled, and a Google account built on Chrome can be used.

**Reply by gz83** [reply]
@whxuezhe Thorium is equipped with the Google synchronization function. If the synchronization cannot be performed due to network problems, you can try several times. The network on my side can use the synchronization function normally.

Also, I would like to ask, does the crash problem you mentioned before still exist?
